lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,152 @@
+# frozen_string_literal: true
+
+class TicTacToe
+    # player tokens
+    X_TOKEN = :X
+    O_TOKEN = :O
+    
```

Comment:
:100: 

I found this explanation on when is better using symbols rather than strings:

> 
> Symbols are meant to be identifiers. If you follow this philosophy, chances are that you will do things right.
> 
> There are several differences between the implementation of symbols and strings. The most important thing about symbols is that they are immutable. This means that they will never have their value changed. Because of this, symbols are instantiated faster than strings and some operations like comparing two symbols is also faster.

[Reference](https://stackoverflow.com/questions/16621073/when-to-use-symbols-instead-of-strings-in-ruby)


And this one:

> You should use symbols as names or labels for things (like methods) & use strings when you care more about the data (individual characters).

[Reference](https://www.rubyguides.com/2018/02/ruby-symbols/)